<launch>
  <arg name="config_file" />
  <arg name="ignore_missing_diagnostics" />
  <arg name="add_leaf_diagnostics" />

  <node pkg="autoware_error_monitor" type="autoware_error_monitor" name="autoware_error_monitor" output="screen">
    <remap from="~input/diag_array" to="/diagnostics_agg" />
    <remap from="~output/driving_capability" to="/autoware/driving_capability" />

    <rosparam command="load" file="$(arg config_file)" />
    <param name="ignore_missing_diagnostics" value="$(arg ignore_missing_diagnostics)" />
    <param name="add_leaf_diagnostics" value="$(arg add_leaf_diagnostics)" />
  </node>
</launch>
